docs/using-data-types.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[ecs-data-types]]
+=== Data Types
+
+IMPORTANT: The limitations noted in this section apply to the OSS-only distribution of the Elastic Stack. The default distribution with the Elastic license supports all data type used by ECS.
```

Comment:
IMO the "default" wording makes more sense inside of Elastic than for the community. I would instead try to use the wording [on the subscriptions page](https://www.elastic.co/subscriptions): namely "Basic", "license", perhaps "tiers".

I would also avoid using the word "limitations". The stack is getting tons of new features, some in OSS, most in Basic for free, and some are in higher tiers only. 

So I'd recommend messaging this with points like this:

* All Elasticsearch data types are available in the free version of the Elastic stack, under the Basic license.
* The Elastic stack downloads are by default Basic license. Unless users specifically downloaded the OSS release, this page does not affect them.
* Some advanced data types are not available in the OSS release of the Elastic Stack.
* ECS now uses basic-licensed data types, and this page covers ways that users of the OSS release can get similar functionality by replacing some of the basic types with somewhat equivalent types available under OSS.

Reply:
Note also that our tech writers or marketing folks could help tweak the messaging as well

Reply:
I took another pass, trying to capture these points better. I agree that we should get some additional 👀 to review the messaging after we're happy with our edits.